src/main/java/org/fcrepo/sword/http/SWORDServiceResource.java

```diff
@@ -28,15 +31,25 @@
  */
 @Scope("request")
 @Path("/sword")
```

Comment:
Using the `@Path` of `/sword` will conflict with the FedoraLdp REST path, and would otherwise pose problems for users who wanted to create a top-level `/sword` repository resource.
Sticking with the convention of `/fcr:sword` would be safe. Would using `/fcr:sword` be reasonable from your perspective?